.ci/Jenkinsfile

```
@@ -60,3 +65,12 @@ def cleanup(){
   }
   unstash 'source'
 }
+
+def reportTestResults(){
+  dir("${WORKSPACE}"){
+    sh "cat ${WORKSPACE}/.elastic-package/tmp/test/reports/"
+  }
+  junit(allowEmptyResults: false,
+    keepLongStdio: true,
+    testResults: "${WORKSPACE}/.elastic-package/tmp/test/reports/*.xml")
```

If no xml files this will fail, if that's what you want, otherwise `allowEmptyResults: true,`
```suggestion
  dir("${WORKSPACE}/.elastic-package/tmp/test/reports"){
    junit(allowEmptyResults: false,
      keepLongStdio: true,
      testResults: "*.xml")
  }
```

---

Yep, `false` is what I want here.

---

My snippet of code what's about using Workspace/base_dir that's where the source code is stored 

If the go junit transformation uses some predefined environmental variable such as TEMP or got some dependencies with the WORKSPACE env variable then that particular location should be reviewed

---

Ah, sorry, I missed that change in your diff. Thanks for pointing it out. The test runner is writing its result files to `~/.elastic-package/tmp/test/results` folder, which in a Jenkins environment corresponds to `$WORKSPACE/.elastic-package/tmp/test/results`. We know this location is valid because the `sh` step that `cat`s the result files is showing their contents as expected: https://beats-ci.elastic.co/blue/organizations/jenkins/Beats%2Felastic-package/detail/PR-136/37/pipeline/#step-116-log-1